package.json

```diff
@@ -15,10 +15,6 @@
     "watch": "babel --watch bin src specs -d dist --source-maps",
     "build-debug": "babel src specs -d dist --source-maps",
     "prepublishOnly": "npm run clean && npm run build",
```

Comment:
@itaymendel remove the comma here, otherwise, it's not a valid json:) and the tests failed